### DISCUSSION TITLE
关于这个 Standard ML

### DISCUSSION THREAD

**Discussion by Young-Lord** [Cute-Dress/Dress#50] [畅聊]
对应的文件是以`.sig`结尾的 GPG 签名，个人感觉这是个错误标注，是否有必要隐藏？具体的分析和做法如下，如果认为可以的话我就开个 pr
<https://github.com/github-linguist/linguist/issues/1298>
<https://github.com/biergaizi/raspberrypi-sources/pull/1>

**Comment by CuteDress** [comment]
看了一下，可以的，你直接开 pr 吧
